### DISCUSSION TITLE
Back up genmon manually

### DISCUSSION THREAD

**Discussion by wrvolz** [jgyates/genmon#1124] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

Latest 

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

My RPI is screwup and I need to reinstall the OS. I want to back up my settings and data but I can't load the genmon webpage to get to the About page. It seems that flask is missing now and the apt system is screwed up enough I can't install anything. Is there a manual way to back things up before I reimage?

**Comment by skipfire** [comment]
You can backup the contents of `/etc/genmon` manually.  I think the command you would use is `tar -zcvf ~/genmon_backup.tar.gz /etc/genmon/`

**Reply by wrvolz** [reply, marked as answer]
Actually in the genmon directory, run "genmonmaint.sh -b -c". This is what genserve.py calls from the About page.

**Reply by jgyates** [reply]
if you use the -c option you must add the full path to the config file folder, if you leave off the -c it will default to /etc/genmon so

      ./genmonmaint.sh -b

will work, or

     ./genmonmaint.sh -b -c /etc/genmon

This will put the file genmon_backup.tar.gz in the ~/genmon folder

**Reply by jgyates** [reply]
https://github.com/jgyates/genmon/wiki/1----Software-Overview#genmonmaintsh
